concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3250,6 +3251,37 @@ jobs:
         params:
           file: updated-tfstate/cf.tfstate
 
+  - name: rotate-cc-db-encryption-key
+    serial: true
+    plan:
+    - aggregate:
+      - get: paas-cf
```

Comment:
We should probably pin this to the version thats passed cf-deploy to ensure we're using a version that matches what's been deployed...

Reply:
Sensible, but paas-cf is only imported for the use of the `./paas-cf/concourse/scripts/bosh_login.sh` script, which is stable; additionally, this pipeline is temporary and will be removed immediately after it is run everywhere. I gave some thought to whether it is worth keeping, but it means speculating about our future implementation of the rotation mechanism.

Reply:
Also, the diff here is weird at the moment because it is a branch off of a branch.